Dockerfile.composer

```diff
@@ -7,9 +7,10 @@ COPY composer.json \
     symfony.lock \
     ./
 
+COPY vendor-extra/ vendor-extra/
+
 RUN composer --no-interaction install ${composer_dev_arg} --ignore-platform-reqs --no-autoloader --no-suggest --prefer-dist
 
 COPY src/ src/
-COPY vendor-extra/ vendor-extra/
```

Comment:
Increase cache busting is a downside. (Though as Flex downloads in parallel it's not that slow.)

Reply:
We could volume the composer cache directory if it becomes a problem I guess